### ISSUE TITLE
No issues found with component header that contains html markup

### ISSUE THREAD

**Issue by jsteinshouer** [cflint/CFLint#652]
I noticed that if a have a cfscript component header with html markup in it that CFlint will not report any issues.

```
/**
 *
 * Some hint
 *
 * <h5>Usage:</h5>
 * <pre>Code example here</pre>
 *
 * 
 */
component {
```

Output:

```
Total files:1
Total lines:57

Issue counts:0

Total issues:0
```

However if I remove the html markup.

```
/**
 *
 * Some hint
 *
 */
component {
```

Then I get the reported issues I expect to see.

```
Total files:1
Total lines:44

Issue counts:3
ARG_HINT_MISSING_SCRIPT:1
FUNCTION_HINT_MISSING:2
MISSING_VAR:2

Total issues:5
Total errors:2
Total infos:3
```

**Comment by KamasamaK** [comment]
I've confirmed this to be a bug in 1.4.1 but looks fine in current 1.5.0-SNAPSHOT.
